src/Stratis.Bitcoin.Features.BlockStore/Controllers/BlockStoreController.cs

```diff
@@ -57,10 +67,22 @@ public async Task<IActionResult> GetBlockAsync([FromQuery] SearchByHashRequest q
             try
             {
                 Block block = await this.blockStoreCache.GetBlockAsync(uint256.Parse(query.Hash)).ConfigureAwait(false);
-                if(block == null) return new NotFoundObjectResult("Block not found");
-                return query.OutputJson 
-                    ? this.Json(new BlockModel(block))
-                    : this.Json(block);
+
+                if (block == null)
+                {
+                    return new NotFoundObjectResult("Block not found");
+                }
+
+                if (!query.OutputJson)
+                {
+                    return this.Json(block);
+                }
+                else
```

Comment:
Maybe you don't need this else

Reply:
This return results in the block being returned as hex. So it is needed, to avoid an API breaking change.

Reply:
Of course, my fault, I have to blame it on a late evening :-) Thanks!